src/blocks/OurTeam/TeamMemberCard.js

```diff
@@ -0,0 +1,65 @@
+import BaseLink from 'components/Link'
+import Image from 'components/Image'
+import styled from 'styled-components'
+import get from 'utils/get'
+import Typography from 'components/Typography'
```

Comment:
```suggestion
import styled from 'styled-components'
import BaseLink from 'components/Link'
import Image from 'components/Image'
import Typography from 'components/Typography'
import get from 'utils/get'
```

You don't need to import React here?